ambry-store/src/test/java/com/github/ambry/store/StorageManagerTest.java

```diff
@@ -357,15 +357,25 @@ public void replicaFromStandbyToInactiveTest() throws Exception {
       assertEquals("Error code doesn't match", StoreNotStarted, e.getErrorCode());
     }
     storageManager.startBlobStore(localReplica.getPartitionId());
-    // 4. success case (verify both replica's state and decommission file)
+    // 4. store is disabled due to disk I/O error
```

Comment:
is there a test for the `"Store " + partitionName + " is not found on current node"` case?

Reply:
test case added